### PR TITLE
Add liveness probe to the flower rc in order to fix a flake

### LINKED ISSUE #27857: [k8s.io] [Feature:Example] [k8s.io] Celery-RabbitMQ should create and stop celery+rabbitmq servers {Kubernetes e2e suite}
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-e2e-gce-examples/12172/

Failed: [k8s.io] [Feature:Example] [k8s.io] Celery-RabbitMQ should create and stop celery+rabbitmq servers {Kubernetes e2e suite}

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/examples.go:189
Expected error:
    <*errors.StatusError | 0xc820807e00>: {
        ErrStatus: {
            TypeMeta: {Kind: "", APIVersion: ""},
            ListMeta: {SelfLink: "", ResourceVersion: ""},
            Status: "Failure",
            Message: "an error on the server has prevented the request from succeeding (get services flower-service)",
            Reason: "InternalError",
            Details: {
                Name: "flower-service",
                Group: "",
                Kind: "services",
                Causes: [
                    {
                        Type: "UnexpectedServerResponse",
                        Message: "Error: 'dial tcp 10.180.3.5:5555: getsockopt: connection refused'\nTrying to reach: 'http://10.180.3.5:5555/'",
                        Field: "",
                    },
                ],
                RetryAfterSeconds: 0,
            },
            Code: 503,
        },
    }
    an error on the server has prevented the request from succeeding (get services flower-service)
not to have occurred
```

Previous issues for this test: #27683 #27726

### PR DESCRIPTION
**Pull request by luxas**
Fixes #27857 
@fejta @ixdy @timstclair @mwielgus

### CODE REVIEW

File: examples/celery-rabbitmq/flower-controller.yaml [hunk 1]
```diff
@@ -18,3 +18,10 @@ spec:
         resources:
           limits:
             cpu: 100m
+        livenessProbe:
+          httpGet:
+            # Path to probe; should be cheap, but representative of typical behavior
+            path: /
+            port: 80
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
```
Comment (ixdy) [inline examples/celery-rabbitmq/flower-controller.yaml]: is 1 second long enough?

**Comment by ixdy** [comment]
I think you'll need to regenerate `test/e2e/generated/bindata.go` for this to have an effect, too, since #25584 is still pending.

cc @jayunit100

File: examples/celery-rabbitmq/flower-controller.yaml (hunk 1 above)
Comment (luxas) [inline examples/celery-rabbitmq/flower-controller.yaml]: Isn't it first waiting 30secs and then queries the endpoint every second after that?
I think that would be fine.

**Comment by luxas** [comment]
How should I regenerate bindata?

File: examples/celery-rabbitmq/flower-controller.yaml (hunk 1 above)
Comment (ixdy) [inline examples/celery-rabbitmq/flower-controller.yaml]: I wasn't sure if a cluster under load might result in this taking more than 1 second. it looks like many of our testing manifests use 5s, though I do see a few other 1s timeouts scattered throughout. not a big deal in any case.

**Comment by ixdy** [comment]
It should really be a script, but the process for regenerating bindata is described in [this comment](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/gobindata_util.go#L22-L36).

**Comment by luxas** [comment]
@ixdy PTAL
If you think this might fix the issue, LGTM it and we'll see if it does :)

**Comment by ixdy** [comment]
lgtm

**Comment by luxas** [comment]
@k8s-bot e2e test this please github issue: #IGNORE

**Pull request merged**